packages/wc-provider/package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "@cardstack/wc-provider",
+  "version": "0.28.15",
+  "private": true,
+  "description": "WalletConnect provider that uses websockets rather than http",
+  "author": "Aierie <michael.khor@cardstack.com>",
+  "homepage": "https://github.com/cardstack/cardstack#readme",
+  "license": "MIT",
```

Comment:
I think this is actually a GPL license (this PR actually solves another issue that we had around separating the WC-provider into it's own package. @lukemelia @ef4 do you guys know of any additional things we need to do in order to indicate this is a GPL license?

Reply:
Yes, this is LGPL-3.0. I wasn't sure if we wanted to handle this in the same PR - or if it should be in the same codebase?

Reply:
it it's just a matter of some boilerplate text we need to include i think it's fine to do here. I vaguely recall @ef4 mentioning that segregating GPL in a mono repo into its own workspace was a thing that other projects have done.

Reply:
WalletConnect has changed to Apache-2.0 as of `1.7.0` so I've changed the files to refer to that in https://github.com/cardstack/cardstack/pull/2544/commits/456a54daf76ff056147c62ce7606b56b150acefe